### PR TITLE
Add Vegard Skogstad to leaderboard 

### PR DESCRIPTION
**Pull request by vskogstad**
Thanks for making this course openly available! 

**Architecture:**
Used mixed precision with bfloat16.
Initialization of projection and classification layers to zero (Like in NanoGPT-speedruns).
Muon with AdamW optimizer. Using same learning rate for both optimizers, and scaling Muon LR following: https://arxiv.org/abs/2502.16982
Layer norm scaling. https://arxiv.org/pdf/2502.05795

**Training:**
Used step_law optimal learning rate/batch size as a starting point. https://arxiv.org/html/2503.04715v6
Scaled up model size to fit within memory while using optimal batch-size. Adjusted model dimensions to get as high MFU as possible. (Meaning prioritize wide matrix-operations over my slow attention implementation and using just 8 attention heads). Ended up going with a lower MFU architecture in the end as it outperformed larger wider models.
Incrementally increased lr from "optimal lr" until I felt I had spent too much on GPU rental. Final run learning rate is 10x step_law optimal, so I am probably doing something wrong in my calculations, or our data sets are very different.

I calculate validation-loss based on the entire validation set. To save training flops, I just do it once at the end of the run.

### CODE REVIEW

**Comment by marcelroed** [comment]
Impressive score! I look forward to validating this.
